### DISCUSSION TITLE
How to feed image as part of the prompt in the task?

### DISCUSSION THREAD

**Discussion by khushalkumar** [crewAIInc/crewAI#1802] [Q&A]
We're using GPT4, which also supports image input. Then why does the agent not understand a base64 image when we provide it? How can we feed the image directly to the agent as part of the prompt in Task?

**Comment by Sk21prince** [comment]
To feed an image directly to an agent as part of the prompt in a Task, you typically need to provide the image URL or a data representation of the image within the text prompt itself, utilizing specific syntax depending on the language model or platform you're using; most commonly, this involves embedding the image data as a special tag or placeholder that the system can interpret as an image input.
